renovate.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["config:base", ":github>ionic-team/ionic-docs>"],
+  "packageRules": [
+    {
+      "packageNames": ["*"],
+      "directoryNames": [
```

Comment:
This is pretty cool! Will this update any and all dependencies, or just certain ones?

Reply:
Also, will these deps be updated in groups or will we get 1 PR per dep?

Reply:
This will update any and all dependencies within the Stackblitz directories. At the moment, it would create a separate PR per dependency.

We could enable [`groupName`](https://docs.renovatebot.com/noise-reduction/#package-grouping) to batch similar dependencies (like `@angular/*`, `@ionic/`, etc.). 

I didn't do it to start, but I would be open to those 2 cases if you agree.

Reply:
It's not required, but I think it would be nice especially when the Angular deps get updated